docs/visualize/heatmap.asciidoc

```diff
@@ -0,0 +1,156 @@
+[[heatmap]]
+== Heat map
+
+Heat maps are graphical representations of data where the individual values are represented as colors.
+
+NOTE: Heat map has been replaced with <<maps>>, which offers more functionality and is easier to use.
+
+[float]
+[[heatmap-aggregation]]
+=== Supported aggregations
+
+Heat maps support metric, sibling pipeline, bucket, and parent pipeline aggregations.
```

Comment:
This information is in the aggregated topic.  Can you link to that instead of repeating here?

Also feels like some of this text should be embedded in the Ui.